### DISCUSSION TITLE
[For UI Contributors] Running the Vue UI in live mode

### DISCUSSION THREAD

**Discussion by samerbahri98** [semaphoreui/semaphore#1739] [Q&A] [answered]
Hello
I am working on the backup config ( #1024 #417 )
I made the necessary changes to the API, which can be seen in the link below
https://github.com/samerbahri98/semaphore/tree/backup_restore
but before I open the pull request I need to make the appropriate changes in the UI.
I need help with running the UI in live server mode so that I can make the changes without having to build with webpack. Is there a way to do that? I can try to fix it on my own if I have to but I am interested to know if there is a solution that is already there and I have failed to notice. By looking at the `Taskfile.yml`, there is a `dc:dev` target but that doesn't seem to fix the issue.

**Comment by tboerger** [comment, marked as answer]
With the change I have commited at https://github.com/ansible-semaphore/semaphore/pull/1741 you are able to run the backend within one terminal and the live reloading dev server in another terminal. Just switch into the web directory and execute `npm run serve`.

**Reply by samerbahri98** [reply]
I tried it now, it's working. thank you :)
